partiql-ast/src/main/java/org/partiql/ast/v1/expr/ExprCollection.java

```diff
@@ -0,0 +1,48 @@
+package org.partiql.ast.v1.expr;
+
+import org.jetbrains.annotations.NotNull;
+import org.partiql.ast.v1.AstNode;
+import org.partiql.ast.v1.AstVisitor;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * TODO docs, equals, hashcode
+ */
+public class ExprCollection extends Expr {
```

Comment:
So that we don't have to switch on the type, we could just create a `BagConstructor` and `ArrayConstructor`?

Reply:
From offline discussion, these other variants will be removed, leaving just bag and array constructors. We will split these into separate classes.